Formula/astro.rb

```diff
@@ -2,9 +2,14 @@ class Astro < Formula
   desc "To build and run Airflow DAGs locally and interact with the Astronomer API"
   homepage "https://www.astronomer.io/"
   url "https://github.com/astronomer/astro-cli/archive/refs/tags/v1.3.0.tar.gz"
-  sha256 "578410a9dcda1797ff223601317afbc636ca546d6de4998c899e3e347b229be4"
+  sha256 "349d1411950d323de0e5af166444c0abdab77ba22bf5c7a3749cc277ab4791d9"
   license "Apache-2.0"
```

Comment:
This won't fix the error, but at least it makes sure your users get the newer version with `brew upgrade`.
```suggestion
  sha256 "349d1411950d323de0e5af166444c0abdab77ba22bf5c7a3749cc277ab4791d9"
  revision 1
  license "Apache-2.0"
```